test/testcafe/spec/TerminalReturnEmail_spec.js

```diff
@@ -0,0 +1,28 @@
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
+import { RequestMock } from 'testcafe';
+import terminalReturnEmail from '../../../playground/mocks/idp/idx/data/terminal-return-email';
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx')
+  .respond(terminalReturnEmail)
+
+fixture(`Return Email Terminal`)
+  .requestHooks(mock)
+
+async function setup(t) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+  await identityPage.fillIdentifierField('Test Identifier');
+  await identityPage.clickNextButton();
+  return new TerminalPageObject(t);
+}
+
+test
+  (`show the correct content`, async t => {
+    const terminalPageObject = await setup(t);
+    await t.expect(terminalPageObject.getHeader()).eql('Email link (o*****m@abbott.dev)');
+    await t.expect(terminalPageObject.getSubtitle()).eql('To finish signing in, return to the screen where you requested the email link.');
+    await t.expect(terminalPageObject.getFooter()).eql('Back to sign in');
```

Comment:
looks like we shall verify 
- the text of back button is `Back to sign in` ?

Reply:
What do you mean exactly? Separate it to a different test?